### PR TITLE
added Is_empty function in paddle.tensor.logic

### LINKED ISSUE #16448: is_empty

### PR DESCRIPTION
**Pull request by JayDeep1710**
Close #16448

### CODE REVIEW

**Comment by JayDeep1710** [comment]
#16448

**Comment by JayDeep1710** [comment]
please review my code. thanks

File: ivy/functional/frontends/paddle/tensor/logic.py
Comment (rishabgit) [inline ivy/functional/frontends/paddle/tensor/logic.py:1]: shouldn't be replacing existing fns and make sure to add a newline at the end

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_logic.py
Comment (rishabgit) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_logic.py:1]: same as above, shouldn't replace existing different tests

**Review by rishabgit — CHANGES_REQUESTED** [review]
Hi! Thanks for working on this 😄 
Looks like you're accidentally removing other functions 😅. Could you please add the is_empty fn as a new function instead of replacing the previous functions?
Apart from this, your PR looks good to me - should be ready to merge after the requested changes 😅😄

**Comment by JayDeep1710** [comment]
i have updated the changes. please check.
Sorry for my silly mistake😅.

**Review by rishabgit — CHANGES_REQUESTED** [review]
Hi! Not sure how I missed this before, but your code seems to call ivy.is_empty which doesn't exist 😅 I'd heavily suggest going through these deep dive resources to help you out - https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html#ivy-frontends, https://unify.ai/docs/ivy/overview/deep_dive/function_types.html
Also, I looked into Paddle's docs - https://www.paddlepaddle.org.cn/documentation/docs/en/2.2/api/paddle/is_empty_en.html#is-empty and the unsupported dtypes needs an update too
Feel free to re-request review after making these changes, thanks! 😄

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**